contracts/SnappAuction.sol

```diff
@@ -185,17 +173,18 @@ contract SnappAuction is SnappBase {
         for (uint i = 0; i < packedOrders.length / 26; i++) {
             orderData = packedOrders.slice(26*i, 26);
 
-            uint8 buyToken = BytesLib.toUint8(orderData, 0);
-            uint8 sellToken = BytesLib.toUint8(orderData, 1);
-
             uint96 buyAmount;
             assembly {  // solhint-disable no-inline-assembly
-                buyAmount := mload(add(add(orderData, 0xc), 2))
+                buyAmount := mload(add(add(orderData, 0xc), 0))
             }
             uint96 sellAmount;
             assembly {  // solhint-disable no-inline-assembly
-                sellAmount := mload(add(add(orderData, 0xc), 14))
+                sellAmount := mload(add(add(orderData, 0xc), 12))
```

Comment:
Are you sure this is correct? We need to introduce a unit test that fetches the event emitted from the transaction and compare that our values are as expected.